tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27
+envlist = py37
```

Comment:
Your travis build is failing because it can't find Python 3.7. You'll need to update `.travis.yml` as well.